frontend/src/container/Login.jsx

```diff
@@ -107,25 +99,25 @@ class Login extends React.Component {
     }
   }
 
-  handleChangeLogin = e => this.setState({ inputLogin: { ...this.state.inputLogin, value: e.target.value } })
-  handleChangePassword = e => this.setState({ inputPassword: { ...this.state.inputPassword, value: e.target.value } })
   handleChangeRememberMe = e => {
     e.preventDefault()
     e.stopPropagation()
     this.setState(prev => ({ inputRememberMe: !prev.inputRememberMe }))
   }
 
-  handleInputKeyDown = e => e.key === 'Enter' && this.handleClickSubmit()
```

Comment:
you removed this feature because you had to but you did not remake it with the new workflow of uncontrolled input

Reply:
I removed it because now the type of form button is `submit` so it can be triggered when `enter` is pressed

Reply:
I not really understand, its no more possible to validate with `ENTER` or `ENTER` works with this specific code?

Reply:
Validating the form with `ENTER` is still working

Reply:
ok, good.